src/constants/route.js

```diff
@@ -23,6 +25,8 @@ const PATH_TITLE = {
   [PATH.SIGN_UP]: 'Sign Up',
   [PATH.ORDERS]: 'Orders',
   [PATH.SIGN_UP_STORE]: 'Sign Up Store',
+  [PATH.STORE_WELCOME]: 'Welcome to Your Store',
+  [PATH.NEED_WALLET]: 'Wallet Providers',
```

Comment:
There are used by the navbar, and are not going to be needed once we take it out. Titles should do directly inside their components.

Reply:
Path titles deleted.